From d074afe174592986aadab61cbc19ac2e079b1aae Mon Sep 17 00:00:00 2001
From: "Gary E. Miller" <gem@rellim.com>
Date: Mon, 8 Oct 2018 18:43:19 -0700
Subject: [PATCH] SConstruct: remove SourceCode()

Scons says:

"warning: SourceCode() has been deprecated and there is no replacement."

Did not seem important, a minor speed-up is lost.
---
 SConstruct | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/SConstruct b/SConstruct
index db61183b3..7ab79bd6d 100644
--- a/SConstruct
+++ b/SConstruct
@@ -450,9 +450,6 @@ if 'CCFLAGS' not in os.environ:
 # We are POSIX 2001, tell the world
 # env.Append(CFLAGS=['-D_POSIX_C_SOURCE=200112L'])
 
-# Get a slight speedup by not doing automatic RCS and SCCS fetches.
-env.SourceCode('.', None)
-
 # Cross-development
 
 devenv = (("ADDR2LINE", "addr2line"),
-- 
GitLab

